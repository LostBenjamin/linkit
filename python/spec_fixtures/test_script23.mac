%include symbols

%run-macro glass-light-white

$brightness 33

[glass-light-white 10]
  (light-set-white)

[glass-light-red 11]
  (light-set-red)

[glass-light-tungsten 12]
  (light-set-tungsten)

[glass-light-neon 13]
  (light-set-neon)

((light-setting white red))
((light-setting red red))
((light-setting tungsten sodium))
((light-setting neon neon))

[[light-setting FRONT-COLOR SIDE-COLOR
  [light-set-FRONT-COLOR]
    (light-side-FRONT-COLOR)
    (light-front-FRONT-COLOR)

  [light-side-FRONT-COLOR]
    SIDE-COLOR
    dim
    dim
    dim
    7
    repeat

  [light-front-FRONT-COLOR]
    FRONT-COLOR
    47
    repeat
]]

