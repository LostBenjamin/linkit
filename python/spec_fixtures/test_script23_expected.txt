10:set:15:run
11:set:9:run
12:set:14:run
13:set:8:run
4:set:neo:47:rep
5:set:red:47:rep
6:set:tun:47:rep
7:set:wht:47:rep
8:set:16:run:4:run
9:set:17:run:5:run
14:set:18:run:6:run
15:set:19:run:7:run
16:set:neo:dim:dim:dim:7:rep
17:set:red:dim:dim:dim:7:rep
18:set:sod:dim:dim:dim:7:rep
19:set:red:dim:dim:dim:7:rep

